java/RSPEC-1174.java

```diff
@@ -0,0 +1,4 @@
+class PublicFinalize {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._